From 708274855501cfb1cf4ed01d4e68758d94a4c506 Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@gmail.com>
Date: Wed, 28 Sep 2022 22:23:34 +0200
Subject: [PATCH] HACK: temporary disable iommu on H6

Since driver is not enabled, peripherals with iommu phandle are probed
very late. Workaround that by removing iommu phandles until driver is
enabled.

Signed-off-by: Jernej Skrabec <jernej.skrabec@gmail.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
index fbe94abbb1f9..4a8abc3ace16 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
@@ -136,7 +136,6 @@ mixer0: mixer@100000 {
 				clock-names = "bus",
 					      "mod";
 				resets = <&display_clocks RST_MIXER0>;
-				iommus = <&iommu 0>;
 
 				ports {
 					#address-cells = <1>;
@@ -171,7 +170,6 @@ video-codec@1c0e000 {
 			resets = <&ccu RST_BUS_VE>;
 			interrupts = <GIC_SPI 89 IRQ_TYPE_LEVEL_HIGH>;
 			allwinner,sram = <&ve_sram 1>;
-			iommus = <&iommu 3>;
 		};
 
 		gpu: gpu@1800000 {
-- 
2.37.3

